quirks/password-rules.json

```diff
@@ -924,7 +924,7 @@
         "password-rules": "minlength: 12; maxlength: 30; required: lower; required: upper; required: digit; required: [!@#$%^&*()+];"
     },
     "umterps.evenue.net": {
-        "password-rules": "minlength: 4; maxlength: 12;"
+        "password-rules": "minlength: 14; required: digit; required: upper; required: lower; required: special;"
```

Comment:
Have you been able to verify that the website supports every character in the `special` set? In my experience, very few websites accept all of them.

Reply:
Good catch, changed the commit to only include special characters accepted by the website!